esmvaltool/cmor/table.py

```diff
@@ -25,18 +25,28 @@ def read_cmor_tables(cfg_developer):
         Parsed config-developer file
 
     """
+    custom = CustomInfo()
+    CMOR_TABLES['custom'] = custom
+
     for table in cfg_developer:
         project = cfg_developer[table]
 
         table_path = project.get('cmor_tables', '')
         table_path = os.path.expandvars(os.path.expanduser(table_path))
-
         cmor_type = project.get('cmor_type', 'CMIP5')
-
+        cmor_strict = project.get('cmor_strict', False)
```

Comment:
`True` would probably be a better default value, as it indicates that this is preferred.